init.lua

```diff
@@ -63,7 +63,15 @@ local function update(widget, reg, disablecache)
 
     local function update_value(data, t, cache)
         if widget.add_value ~= nil then
-            widget:add_value(tonumber(data) and tonumber(data)/100)
+            if widget.get_stack and widget:get_stack() then
+                for idx, col in ipairs(widget:get_stack_colors()) do
+                    if data[idx] then
+                        widget:add_value(tonumber(data[idx])/100, idx)
```

Comment:
Shouldn't we check if `tonumber` have succeeded, i.e. `widget:add_value(tonumber(data[idx]) and tonumber(data[idx]/100), idx)`?

Please edit the patch to be compatible with current master and update the [README example on stacked graph](https://github.com/vicious-widgets/vicious/blob/065bd1d0097d6bdc737a2780fc469dd6332b0f5d/README.md#format-functions) before merging.